ydb/public/sdk/cpp/client/ydb_federated_topic/impl/federated_write_session.cpp

```diff
@@ -122,22 +124,20 @@ void TFederatedWriteSessionImpl::OpenSubsessionImpl(std::shared_ptr<TDbInfo> db)
     auto subclient = make_shared<NTopic::TTopicClient::TImpl>(Connections, clientSettings);
 
     auto handlers = NTopic::TWriteSessionSettings::TEventHandlers()
-        .HandlersExecutor(Settings.EventHandlers_.HandlersExecutor_)
+        .HandlersExecutor(NTopic::CreateThreadPoolExecutor(1))
```

Comment:
This must be one executor per driver. What will happen if client creates 1000 federated write sessions? This code will create 1000 threads.